### ISSUE TITLE
problems on nodejs v4.2.1 LTS.

### ISSUE THREAD

**Issue by ezekkiel** [meanjs/mean#1015]
Have problems with karma on ```npm install``` with nodejs v4.1.2 : 

```
npm WARN engine karma@0.12.37: wanted: {"node":">=0.8 <=0.12 || >=1 <=2"} (current: {"node":"4.1.2","npm":"2.14.4"})
npm WARN peerDependencies The peer dependency istanbul@0.x.x included from grunt-mocha-istanbul will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
```

**Comment by codydaig** [comment]
We don't fully support Node 4 yet. The master branch has some initial support for Node 4, but we don't fully support it yet. We're waiting on all dependencies to catch up, and for us to be able to test and bring them in.

**Comment by ezekkiel** [comment]
Ok, Thanks!, would be nice to add to that to the main page. Cheers.

**Comment by ilanbiala** [comment]
@ezekkiel those are easy fixes and actually, a PR would be very welcome. 1 PR would be to update our dependency of Karma to the latest. Another PR would be to add Istanbul as a devDependency, which will get rid of those warnings.

**Comment by codydaig** [comment]
@ezekkiel If you use the master branch of this repo, it does work on Node 4. I've been using it for several weeks now. There are npm WARN messages, but those are just warnings for when npm v3 is released. a PR is still welcome, but not necessary to get it working on Node 4.

**Comment by lirantal** [comment]
@codydaig so we can close it for now then. If problems arise with Node v4 on the latest master branch then we can re-open.

**Comment by ilanbiala** [comment]
@lirantal this should actually be fixed before we start using npm 3, so I'm going to leave it open and mark it as 0.5.0 if @ezekkiel doesn't submit a PR by then.

**Comment by lirantal** [comment]
Ok

**Comment by palpalani** [comment]
There is new package for checking node 4.2 compatibily -  https://www.npmjs.com/package/upgrade-ready.
Here is the result for master branch - http://upgrade-ready.nodesource.com/summary/adac5b50-8885-11e5-b0d6-136ad07fd6ea
